products/green_fast_track/models/intermediate/spatial/shadows/int_spatial__shadow_nat_resources.sql

```diff
@@ -1,15 +1,8 @@
 WITH all_natural_resources AS (
 {{ dbt_utils.union_relations(
     relations=[
-        ref('stg__nysdec_freshwater_wetlands'),
-        ref('stg__nysdec_tidal_wetlands'),
-        ref('stg__nysdec_priority_lakes'),
-        ref('stg__nysdec_priority_estuaries'),
-        ref('stg__nysdec_priority_streams'),
-        ref('stg__dcp_beaches'),
-        ref('stg__nysdec_natural_heritage_communities'),
-        ref('stg__usfws_nyc_wetlands'),
-        ref('stg__dpr_forever_wild'),
+        ref('int_spatial__natural_resources'),
+        ref('int_flags__dob_natural_resources'),
```

Comment:
I think this should only be `int_spatial__natural_resources` since `int_flags__dob_natural_resources` should be going into that right?

Reply:
It is not - see comment here https://github.com/NYCPlanning/data-engineering/pull/878/files/0c8c7a0a949ac627251f4e3ee9b1458df5425781#diff-c227eefe4b635026c8543a11f3cde0b647b9957f322567a2911c3850997268c9R7

Reply:
Basically, I didn't add it to int_spatial__natural_resources because it doesn't need a spatial join to determine flags since they're provided tabularly

Reply:
that makes sense. then `int_flags__dob_natural_resources` shouldn't be in this model too then right?

it's used in `int_flags__all`

Reply:
It should be in this model - Jack and Matt were both explicit that they want the lots buffered as part of the shadow dataset. 

So 
- plain natural resources - just join bbl
- shadows - join buffered bbl lot geom

Reply:
ahhh ok thank you. so either a project includes on of these lots or it's "shadow adjacent" to one

Reply:
Precisely